### DISCUSSION TITLE
as company sponsor, how org's member do sync fork.

### DISCUSSION THREAD

**Discussion by samzong** [squidfunk/mkdocs-material#6214] [General]
Hi @squidfunk,

I'm an member in Org, and we have sponsor you as company, but I can not do sync fork in fork repo.

According to the prompts on the application for sponsorship page, I have sent an email requesting sponsors@squidfunk.com, from yongjie.xu at daocloud.io to apply to add our member github id.

at now, I also can not sync fork.

**Comment by squidfunk** [comment]
Hey @samzong. Thanks for reaching out. I'm sorry that we haven't reached out to you, it's because your email went to spam. It might be a good idea to check why this is the case. We'll be in touch shortly.

**Reply by samzong** [reply]
Okay, I used a company email account, and I'll send this email again using my Gmail.
